src/Behat/Behat/Output/Node/Printer/Html/AbstractHtmlPrinter.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Behat.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Behat\Output\Node\Printer\Html;
+
+use Behat\Behat\Output\Node\Printer\Helper\HtmlPrinter;
+use Behat\Testwork\Output\Printer\OutputPrinter;
+
+/**
+ * To retrieve (html printer) helper .
+ *
+ * @author Ali Bahman <abn@webit4.me>
+ */
+abstract class AbstractHtmlPrinter
```

Comment:
I took that abstraction out but i have to admit now there is too much duplication in such small module :) 
anyway all changes are in a single commit so we would be able to revert them if change our mind